esrfconcert/experiments/laminography.py

```diff
@@ -64,6 +64,8 @@ def __init__(self, walker, flat_motor, scanning_motor, shutter, radio_position,
                                                      camera=camera, num_flats=num_flats, num_darks=num_darks,
                                                      num_projections=num_projections, angular_range=angular_range,
                                                      start_angle=start_angle, separate_scans=separate_scans)
+        self['radio_position']._parameter.unit = q.deg
+        self['flat_position']._parameter.unit = q.deg
```

Comment:
@MarcusZuber I had a look in ankaconcert and you "overwrote" these in the mixin class. We should think of a clean way how to deal with these.